src/Driver/Result.php

```diff
@@ -88,6 +86,11 @@ public function rowCount(): int|string;
      */
     public function columnCount(): int;
 
+    /**
+     * Returns the name of the column in the result set for the given 0-based index.
```

Comment:
you should probably add the `@throws Exception` for consistency with other methods

Reply:
Good idea. If you don't send a PR, I'll do that tomorrow. 🙂